src/main/scala/org/broadinstitute/dsde/firecloud/model/ModelSchema.scala

```diff
@@ -22,22 +22,6 @@ object ModelSchema {
       case None => Failure(new RuntimeException("Unknown entity type: " + entityType))
     }
   }
```

Comment:
This `getTypeSchema` is the only thing remaining in `ModelSchema` and is the only used in one other place (`backwardsCompatStripIdSuffixes(...)`) - is necessary to keep it in a separate object?